.github/workflows/staging-daily.yml

```diff
@@ -69,6 +69,9 @@ jobs:
       - name: Bugzilla Desktop Bugs
         run: python ./__main__.py --report-type bugzilla-desktop-bugs
 
+      - name: Bugzilla Release Flags for Bugs
+        run: python ./__main__.py --report-type bugzilla-desktop-release-flags-for-bugs
```

Comment:
Leaving this here for now to debug it when it lands, it would make sense to run it weekly better